portfolio/src/main/webapp/charts.html

```diff
@@ -13,6 +13,9 @@
     <div class="content">
       <h1>Rice Supplied (g/person/day) by Country</h1>
       <div id="chart-container"></div>
+      <div class="buttons-row">
```

Comment:
I think the button changes should be in their own PR, but I'm fine with keeping them in the current one for simplicity's sake. Good idea to boil these down to plain anchor tags.